test/multifile/constant-struct-with-padding/Other.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: true
```

Comment:
I don't know whether it is needed or not, I just cargo-culted/copied it from the other tests in `test/multifile`.